pagopa/pn-frontend
# Pn 5085 fe inserimento notifica con numero di protocollo duplicato

## Short description
Added new error handler about protocol number duplcated.

## List of changes proposed in this pull request
- Added Invalid Parameter Duplicate handler in ApiError
- Added tests

## How to test
1. Create a new notification, take note of protocol number.
2. Create another notification with the same protocol number.
3. You should get an error about existing (duplicated) protocol number now.
